README.md

```diff
@@ -46,10 +46,27 @@ unleash.on('synchronized', () => {
   const variant = unleash.getVariant('app.ToggleY');
 });
 ```
+Be aware that the `initialize` function will configure a _global_ Unleash instance. If you call this method multiple times the global instance will be changed. 
 
-Be aware that the `initialize` function will configure a _global_ Unleash instance. If you call this
-method multiple times the global instance will be changed. If you prefer to handle the instance
-yourself you should [construct your own Unleash instance](#alternative-usage).
+#### <u>Constructing the Unleash client directly</u>
+If you prefer to construct the Unleash instance yourself rather than using the global `initialize` method you can do so e.g.:
+```js
+const { Unleash } = require('unleash-client');
+
+const unleash = new Unleash({
+  appName: 'my-app-name',
+  url: 'http://unleash.herokuapp.com',
+  customHeaders: {
+    Authorization: 'API token',
+  },
+});
+
+unleash.on('ready', console.log.bind(console, 'ready'));
+
+// required error handling when using unleash directly  
+unleash.on('error', console.error);
+```
+**_IMPORTANT:_** When using unleash directly, error handling is required by adding an event listener for the error event. Otherwise your application may either not start up, or crash if its running when the application can't reach the unleash server.
```

Comment:
This is great! But I think it might make sense to move it to before the code sample, so that it's:
1. introductory paragraph
2. error handling
3. code sample

What do you think? ☺️ 

Also a minor rewording suggestion:

```suggestion
**Important:** When Using unleash directly, you **must** handle errors yourself. This can be done by attaching an event listener to the `error` event, as shown in the example below. If you don't do this, your application may crash either on startup or at runtime if it can't reach the Unleash server or if it encounters other errors.
```

Reply:
> This is great! But I think it might make sense to move it to before the code sample, so that it's:
> 
> 1. introductory paragraph
> 2. error handling
> 3. code sample
> 
> What do you think? ☺️
I agree will change the order of things 👍🏽

Reply:
> 
> Also a minor rewording suggestion:
Thanks for the rewording suggestion, gave this text a couple of rounds but thought that it lacked something that's what PR's for 🙂  it reads much better now 👍🏽